packages/cx/src/widgets/form/LookupField.js

```diff
@@ -720,6 +720,20 @@ class LookupComponent extends VDOM.Component {
             e.stopPropagation();
             break;
 
+         case KeyCode.enter:
```

Comment:
I'm not a big fan of switch case fall through. Can you please add else openDropdown and a break?